openmc/material.py

```diff
@@ -914,6 +981,52 @@ def to_xml_element(self, cross_sections=None):
 
         return element
 
+    @classmethod
+    def volume_mix(materials,fracs,name=None):
+        """Mix multiple materials together based on volume fractions
+
+        Parameters
+        ----------
+        materials : Iterable of openmc.Material
+            Materials to combine by volume
+        fracs : Iterable of float
+            Fractions of each material to be combined 
+
+        """
+        fracs = np.asarray(fracs)
+        if np.sum(fracs) != 1:
+            # Issue warning that fractions don't add to 1 set remainder to void
+            print("Warning: sum of volume fractions do not add to 1, \
+                  void fraction set to {}".format(1-np.sum(fracs)))
+        if name is None:
+            name = "-".join(["{0}({1})".format(m.name,f) for m,f in zip(materials,fracs)])
```

Comment:
Follow [PEP8](https://www.python.org/dev/peps/pep-0008/) for Python code
```suggestion
            name = "-".join(["{}({})".format(m.name, f) for m, f in zip(materials, fracs)])
```